src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,97 @@
-import React from 'react';
+import React, { useMemo } from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[];
+  select: string;
+  query: string;
+  handlerModButton: (todo: Todo) => void;
```

Comment:
Consider renaming `handlerModButton` to `handleModButtonClick` to follow common naming conventions for event handlers and improve code readability.